tests/test_features/test_pcoords.py

```diff
@@ -163,11 +170,12 @@ def test_pandas_integration_sampled(self):
         """
         Test on a real dataset with pandas DataFrame and Series sampled for speed
         """
-        df = self.load_pandas("occupancy")
+        data = load_occupancy(return_dataset=True)
+        df = data.to_dataframe()
```

Comment:
Awesome! This is exactly what I hoped would happen with the pandas based tests! This ensures that it's a data frame. However, this brings up another issue, do we need to also ensure that we test numpy arrays, e.g. `test_numpy_integration_sampled` with `data.to_numpy()`? Using `load_occupancy()` will return a DataFrame if pandas is installed, otherwise, it will return numpy arrays. This could cause problems with the image similarity tests. What do you think we should do here?

Reply:
If the main problem is with the text in image similarity tests, then we can test the texts separately. But I'm actually not sure how to use the same test function for both pandas DataFrame and numpy array because the features are selected based on column name.

What if we have another test function that runs only when `pd is None`?

Reply:
It's not about text so much as my wanting to ensure that our visualizers work both for numpy and pandas. Previously everything was numpy arrays and we did something special for pandas. But with the new datasets module, if pandas is installed the tests will be all pandas and no numpy. 

It's a good idea to have a test that only runs when `pd is not None` to specifically test that case for numpy!